test/user.test.js

```diff
@@ -75,20 +76,23 @@ describe('user routes', () =>{
             const res2 = await request.get('/api/users');
             expect(res2.body[3].preferredPaymentMethod).toBe('CASH')
         })
+
         // //ATM UUID is passed in with making a post request, but should change this so a UUID is generated for the user
-        it('should throw an error if uuid, username, password is omitted', () =>{
-                try{  
-                    const res = request
-                    .post('/api/users')
-                    .send({
-                        uuid: '6',
-                        password: 'testPassF'
-                    })
-                }
-                catch(err){
-                    expect(err.message).toBe('userName required')
-                }
-        })
+        // it('should throw an error if uuid, username, password is omitted', () =>{
```

Comment:
Why is this commented out? Is it because of the UUID that should be implemented?

Reply:
As per my PR comment, this test doesn't exit properly because the request isn't getting resolved due to the error we're throwing.

Reply:
got it, I'll take a look at this test to see what the issue may be

Reply:
@elliely32 All gucci, you can focus on your current sprint taks for now. If the investigation/fix is still WIP, you may take over/pick it up.